soccer/Robot.hpp

```diff
@@ -530,6 +532,9 @@ class OurRobot : public Robot {
     friend class MotionControl;
 
 private:
+    mutable RJ::Time _lastBallSense;
```

Comment:
Hmm, it might be better just to make hasBall non-const now. I'm not sure if this or the const behavior is better style though......